### PR TITLE
Add RELEASE.md runbook

### PR DESCRIPTION
**Pull request by mxsdev**
Proto commands tested on latest changes

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+## Protos
+
+To release protobuf definitions:
+
+``` sh { name=buf-token }
+export BUF_TOKEN=Your buf token
+```
+
+``` sh { name=release-buf }
+cd ./internal/api && buf push
```
Comment (sourishkrout) [inline RELEASE.md:12]: I believe we have a make target for this, no?

**Review by sourishkrout — CHANGES_REQUESTED** [review]

File: RELEASE.md (hunk 1 above)
Comment (mxsdev) [reply RELEASE.md]: Oh yeah, my bad

File: RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Release
+
+## Protos
+
+To release protobuf definitions:
+
+``` sh { name=buf-token }
```
Comment (sourishkrout) [inline RELEASE.md:7]: The space will break execution.

**Review by sourishkrout — CHANGES_REQUESTED** [review]
See comment

**Review by sourishkrout — APPROVED** [review]
works 👍

**Pull request merged**